src/library/formik/formikHandleInputTypes.js

```diff
@@ -0,0 +1,31 @@
+import { getNumberTruncatedToXDecimalPlaces } from '../numbers/getNumberTruncatedToXDecimalPlaces'
+
+export const formikHandleNumericDecimalInputChange = ({
+  formik,
+  event,
+  fieldName,
+  maxNumberOfDecimals = 2,
+}) => {
+  formik.setFieldValue(
+    fieldName,
+    getNumberTruncatedToXDecimalPlaces({
+      inputValue: event.target.value,
+      maxNumberOfDecimals,
+    }),
+  )
```

Comment:
this is actually a renamed file. The lines hilighted here were reduced from many many more and put into a reusable function (that I can unit test which is easier than manual dev testing in this case)